config/prod.exs

```diff
@@ -17,6 +17,15 @@ config :wanda, WandaWeb.Endpoint,
 # Do not print debug messages in production
 config :logger, level: :info
 
+config :wanda, Wanda.Messaging.Adapters.AMQP,
```

Comment:
We don't need to change the prod env, right?
It is already inheriting this exact values from config

Reply:
yep exactly. I like to keep the config with "standard" values and the other env as an override of them.

Reply:
But it is simply overriding the same values XD
So it doesn't do anything at the end.